src/components/TransactionalInfo.tsx

```diff
@@ -76,10 +75,17 @@ const TransactionalInfo = ({ transaction: tx, addressHash, className, hideLeftAd
   if (!_addressHash) return null
 
   if (isExplorerTransaction(tx)) {
-    amount = calculateAmountSent(tx)
-    direction = getDirection(tx, _addressHash)
-    infoType =
-      !hideLeftAddress && direction === 'out' && hasOnlyOutputsWith(tx.outputs ?? [], addresses) ? 'move' : direction
+    if (isConsolidationTx(tx) && tx.outputs) {
+      amount = tx.outputs.reduce((acc, output) => acc + BigInt(output.attoAlphAmount), BigInt(0))
+      direction = 'out'
+      infoType = 'move'
+    } else {
+      amount = calAmountDelta(tx, _addressHash)
+      direction = amount < 0 ? 'out' : 'in'
+      amount = amount < 0 ? amount * BigInt(-1) : amount
+      infoType =
+        !hideLeftAddress && direction === 'out' && hasOnlyOutputsWith(tx.outputs ?? [], addresses) ? 'move' : direction
+    }
```

Comment:
Could we extract this as a function? It's basically reused in `src/modals/TransactionDetailsModal.tsx`

Reply:
Already done!